deploy/README.md

```diff
@@ -333,6 +327,16 @@ It is possible to migrate an existing Wikibase installation to WBS Deploy. The g
  - Regenerate the WDQS database
  - Regenerate the Elasticsearch database
 
+### Can I host WBS Deploy locally?
+
+Due to the OAuth configuration for MediaWiki and QuickStatements, along with the automatic SSL certification generated by Traefik, you must specify values for `WIKIBASE_PUBLIC_URL` and `QUICKSTATEMENTS_PUBLIC_URL` in your `.env` file. These values should be domain names that route to the IP address of the server hosting these services and must be accessible on the Internet.
+
+However, for testing purposes WBS Deploy can be run locally on a server that is not accessible to the Internet, with the following caveats:
```

Comment:
```suggestion
However, for testing purposes WBS Deploy can be run locally on a system that is not accessible to the Internet, with the following caveats:
```

Reply:
Ha, I went back and forth on that as well. I concluded that "server" makes sense as the system is implicitly operating in the role of a server when running Docker images, and we use that word in other places in the doc (except in the intro paragraph). This could maybe should be normalised across the doc and I would be equally happy with "server", but wasn't taking that on here.

Reply:
Yeah, good point. I feel I am still missing some clarity in the local/private vs server/public discussion.